.github/workflows/ci.yaml

```diff
@@ -112,8 +116,43 @@ jobs:
           restore-keys: ${{ runner.os }}-go-
       - name: Build All Binaries
         run: make build
+
+  detect-noop:
```

Comment:
Could you please give a little more information of the `detect-noop` job about why we need it? It seems to have nothing to do with the multi-k8s ci.